### PR TITLE
CeramicStudio/Webplayground #30 - Add IDX Quickstart

### PR DESCRIPTION
**Pull request by Sterahi**
Created a quickstart guide to give tangible experience from grabbing an existing record to creating a new schema & a record using the newly created schema.

### CODE REVIEW

File: docs/tools/idx/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    :octicons-alert-16: IDX is in alpha, Libraries may be unstable and APIs are subject to change. Data created on IDX during alpha will *not* be portable to production. Please share what you're working on and report any issues in the [IDX Discord](https://chat.idx.com){:target="_blank"}.
+
+!!! warning ""
+    :octicons-stop-16: If you haven't completed the Ceramic quickstart some information here may not make complete sense. We recommend coming here after you've completed the [Ceramic JavaScript Client Quickstart](../../build/javascript/quick-start.md).
+
+## **Prerequisites**
+
+The IDX SDK requires [Node.js](https://nodejs.org/){:target="_blank"} v14+ and npm v6+ (usually installed with Node.js). Make sure both are installed.
+
+On Linux you will also need the `libsecret` library to be installed, as [instructed here](https://github.com/atom/node-keytar#on-linux).
```
Comment (PaulLeCam) [inline docs/tools/idx/quickstart.md:16]: The `libsecret` dependency is specific to the CLI, it shouldn't be needed when using the library.

File: docs/tools/idx/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+=== "Command"
+
+    ```JavaScript
+    await idx.get('basicProfile')
+    ```
+
+=== "Output"
+
+    ```JSON
```
Comment (PaulLeCam) [inline docs/tools/idx/quickstart.md:87]: Output should be JS here when using the library.

File: mkdocs.yml [hunk 3]
```diff
@@ -184,3 +185,4 @@ plugins:
             'tools/identity/identitylink.md': 'tools/identitylink/overview.md'
             'tools/identity/self-id.md': 'tools/self-id/overview.md'
     - search
+{"mode":"full","isActive":false}
```
Comment (PaulLeCam) [inline mkdocs.yml:188]: Is this expected? Seems strange to have a JSON object in a YAML file.

File: docs/tools/idx/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+You will need to connect to a Ceramic node. The following is an example of connecting to a our Clay TestNet.
+
+=== "Command"
+
+    ```JavaScript
+    import CeramicClient from '@ceramicnetwork/http-client'
+
+    const API_URL = 'https://gateway-clay.ceramic.network'
+
+    const ceramic = new CeramicClient(API_URL)
```
Comment (PaulLeCam) [inline docs/tools/idx/quickstart.md:35]: There should be an extra line to create the IDX instance, no?

File: docs/tools/idx/quickstart.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    ```bash
+    npm i @ceramicstudio/idx @ceramicnetwork/http-client
+    ```
+
+You will need to connect to a Ceramic node. The following is an example of connecting to a our Clay TestNet.
+
+=== "Command"
+
+    ```JavaScript
+    import CeramicClient from '@ceramicnetwork/http-client'
```
Comment (PaulLeCam) [inline docs/tools/idx/quickstart.md:31]: The IDX lib should also be imported here, no?

File: docs/tools/idx/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
+# Quickstart
+
+Get started exploring what's possible with IDX using the [IDX SDK](../reference/idx.md). For usage in a Command Interface, you will need to [install the IDX CLI](../reference/cli.md).
```
Comment (stbrody) [inline docs/tools/idx/quickstart.md:3]:
```suggestion
Get started exploring what's possible with IDX using the [IDX SDK](../reference/idx.md). For usage in a Command Line Interface, you will need to [install the IDX CLI](../reference/cli.md).
```

File: docs/tools/idx/quickstart.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+!!! warning ""
+
+    :octicons-alert-16: IDX is in alpha, Libraries may be unstable and APIs are subject to change. Data created on IDX during alpha will *not* be portable to production. Please share what you're working on and report any issues in the [IDX Discord](https://chat.idx.com){:target="_blank"}.
```
Comment (stbrody) [inline docs/tools/idx/quickstart.md:7]: woah. Do our users building production apps on IDX know that we're not guaranteeing their profiles will persist going forward?

File: docs/tools/idx/quickstart.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+!!! warning ""
+    :octicons-stop-16: If you haven't completed the Ceramic quickstart some information here may not make complete sense. We recommend coming here after you've completed the [Ceramic JavaScript Client Quickstart](../../build/javascript/quick-start.md).
+
+## **Prerequisites**
+
+The IDX SDK requires [Node.js](https://nodejs.org/){:target="_blank"} v14+ and npm v6+ (usually installed with Node.js). Make sure both are installed.
```
Comment (stbrody) [inline docs/tools/idx/quickstart.md:14]:
```suggestion
The IDX SDK requires [Node.js](https://nodejs.org/){:target="_blank"} v14 and npm v6 (usually installed with Node.js). Make sure both are installed.
```

Comment (stbrody) [reply docs/tools/idx/quickstart.md:14]: I'm pretty sure today Ceramic only works on *exactly* node v14 and npm v6.  Pretty sure we've had issues reported by the community using newer versions.

File: docs/tools/idx/quickstart.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+## **Step 1: Install the IDX SDK**
+
+=== "Command"
+
+    ```bash
+    npm i @ceramicstudio/idx @ceramicnetwork/http-client
+    ```
+
+You will need to connect to a Ceramic node. The following is an example of connecting to a our Clay TestNet.
```
Comment (stbrody) [inline docs/tools/idx/quickstart.md:26]:
```suggestion
You will need to connect to a Ceramic node. The following is an example of connecting to the Clay Testnet via the public gateway node hosted by 3BoxLabs.
```

Comment (stbrody) [reply docs/tools/idx/quickstart.md:26]: I don't think I've ever seen the "n" in Testnet capitalized before?  Not 100% confident on that though

File: docs/tools/idx/quickstart.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+    ```JavaScript
+    // Note: this record is live and may not return exactly what you see below.
+    {
+      name: "Paul",
+      residenceCountry: "IE"
+    }
+    ```
+
+## **Step 3: Create a DID**
+The IDX SDK does not support DID Creation, however we've built [3ID-Connect](../../authentication/3id-did/3id-connect.md) to allow web apps to create a DID based off of your wallet.
```
Comment (stbrody) [inline docs/tools/idx/quickstart.md:61]:
```suggestion
The IDX SDK does not support DID Creation, however we've built [3ID-Connect](../../authentication/3id-did/3id-connect.md) to allow web apps to create a DID based off of the user's blockchain wallet.
```

**Review by stbrody — COMMENTED** [review]
Looking good!   A few small comments throughout

File: mkdocs.yml (hunk 3 above)
Comment (Sterahi) [reply mkdocs.yml:188]: This is not expected. For some reason when I was copying from github's raws. Good catch though!

File: docs/tools/idx/quickstart.md (hunk 7 above)
Comment (Sterahi) [reply docs/tools/idx/quickstart.md:7]: This is a direct copy from our other IDX page. I'm not certain on if this warning is more of a Ceramic TestNet warning or if it's an IDX problem. @PaulLeCam could you weigh in?

File: docs/tools/idx/quickstart.md (hunk 8 above)
Comment (Sterahi) [reply docs/tools/idx/quickstart.md:14]: I think that it works on other versions, but it needs some extra config. I'm not opposed to making the change regardless and having it a bit more specific will minimize the questions users have.

File: docs/tools/idx/quickstart.md (hunk 9 above)
Comment (Sterahi) [reply docs/tools/idx/quickstart.md:26]: That might just be a me thing. Reasonable grammar fixes though.

File: docs/tools/idx/quickstart.md (hunk 7 above)
Comment (PaulLeCam) [reply docs/tools/idx/quickstart.md:7]: We've had this messaging in the docs for as long as I can remember, mostly warning that until IDX reaches a v1 version there can be breaking changes.

Comment (Sterahi) [reply docs/tools/idx/quickstart.md:7]: Alright, we'll keep it then.

File: mkdocs.yml (hunk 3 above)
Comment (stbrody) [reply mkdocs.yml:188]: FYI this is still showing up in the current version of the PR for me

**Review by stbrody — APPROVED** [review]
I'm happy when Paul and Sena are

**Review by PaulLeCam — APPROVED** [review]
LGTM except the extra JSON in the YAML file.

**Pull request merged**